storage/SystemInfo/CmdDmsetup.h

```diff
@@ -94,31 +94,38 @@ namespace storage
 
 	CmdDmsetupTable();
 
-	// TODO this will likely need to be extended to provide more table-data, e.g. the type
-
-	struct Entry
+	struct Table
 	{
-	    Entry() : majorminors() {}
+	    Table(const string& target)
+		: target(target), stripes(0), stripe_size(0), majorminors() {}
+
+	    string target;
```

Comment:
Oh. I thought there would be just a handful, in which case we could list them in a comment.